src/hash.cr

```diff
@@ -171,6 +171,21 @@ class Hash(K, V)
     indexes.map {|index| self[index] }
   end
 
+  # Returns the corresponding key with the given *value*.
```

Comment:
How about first instead of corresponding? Since a hash can have the same value multiple times.